src/main.ts

```diff
@@ -20,13 +21,14 @@ export async function run(): Promise<void> {
 
     if (graphql.length !== 0) {
       method = 'POST'
-      data = graphqlPayloadFor(graphql)
+      data = graphqlPayloadFor(graphql, variables)
```

Comment:
Why do you assume that it will be a POST if there is a graphql input ?

Reply:
good catch! I should've documented this design choice:

The GraphQL design spec states that we can send query requests via GET. In order to save myself some time, I chose to implement just POST, because it supports both queries and mutations.

This is a good enhancement that we could work on: support both GET and POST GraphQL requests. I added it to our board in case you wanna take a bit at it.